hercules-ci.nix

```
@@ -17,5 +17,23 @@
     };
   };
 
-  herculesCI.ciSystems = [ "x86_64-linux" "x86_64-darwin" ];
+  herculesCI = herculesArgs: {
+    onPush.plutus-ledger-api-publish = {
+      outputs.effects = withSystem "x86_64-linux"
+        ({ hci-effects, config, ... }:
+          hci-effects.runIf
+            (herculesArgs.config.repo.branch != null && (builtins.match "v[0-9]+" herculesArgs.config.repo.branch) != null)
```

Not necessarily wrong, because I don't know your release process, but I think you're looking for a tag.

```suggestion
            (herculesArgs.config.repo.tag != null && (builtins.match "v[0-9]+" herculesArgs.config.repo.tag) != null)
```

---

We plan to have release branches, one for each major version, so the downstream packages can follow these and use hercules auto-flake update without breaking (given that there're no breaking changes in major versions)

---

I actually decided to change this, we want tags after all for all releases, so why not make it part of the workflow.